app/views/fact_values/_fact.html.erb

```diff
@@ -24,7 +24,7 @@
   <td class='fact-origin'><%= fact_origin_icon(fact.origin) %></td>
   <td><%= fact_from fact %></td>
   <td><% unless fact.compose %>
-          <%= link_to_function(content_tag(:button, icon_text("adjust", "")),
+          <%= link_to_function(content_tag(:span, "Distribution Chart", :class => "btn btn-xs btn-default", :style => "font-size: 10px;"),
```

Comment:
this is very specific adjustment because the button text here is too long, I think usually there is no need for that
isn't inline style exactly for such cases? should we create a CSS class just for that? 
But anyway with "View Chart" no need for this adjustment